example/src/main/java/com/appunite/rx/example/MainActivity.java

```diff
@@ -92,5 +105,17 @@ public void call(MainPresenter.AdapterItem adapterItem) {
             }
         };
     }
+    @Nonnull
+    private static Action1<Object> startPostActivityAction(final Activity activity) {
+        return new Action1<Object>() {
+            @Override
+            public void call(Object o) {
+                final Bundle bundle = ActivityOptionsCompat.makeSceneTransitionAnimation(activity)
+                        .toBundle();
+//                ActivityCompat.startActivity(new Intent(activity, )
```

Comment:
- Commented line
- you don't use bundle
- use CreatePostActivity.newIntent(activity)